main/src/library/Benchmark/Bench.Fixpoint.flix

```diff
@@ -149,3 +149,8 @@ namespace Bench/Fixpoint {
         ]
 
 }
+
+def main(_args: Array[String]) : Int32 & Impure =
```

Comment:
I think this might break the arewefast.flix.dev, so I think it has to be removed (or you can comment it, that is okay too)